test/models/post_test.rb

```diff
@@ -26,7 +26,6 @@ class PostTest < ActiveSupport::TestCase
   should validate_presence_of(:body)
   should validate_presence_of(:kind)
   should validate_presence_of(:user)
-  should validate_length_of(:body).is_at_most(10_000)
```

Comment:
@CGA1123 I intended to leave it in, but it fails now that we have the truncation. That test tries to set the body to 10,0001 "x"s:

```
Expected errors to include "is too long (maximum is 10000 characters)" when body is set to [insert 10,001 "x"]
```

Is there a way to bypass the `truncate_body` method for that test?